dbsnp-import/src/main/java/uk/ac/ebi/eva/server/controllers/DbsnpRestController.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2019 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.server.controllers;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import uk.ac.ebi.eva.server.models.ProgressReport;
+import uk.ac.ebi.eva.server.repositories.ProgressReportRepository;
+
+@RestController
+@RequestMapping(value = "/v1/import-status", produces = "application/json")
+public class DbsnpRestController {
```

Comment:
Sorry if this is being super-picky, but maybe this could be renamed as `DbsnpProgressReportRestController`? Just to make more clear which dbSNP-related aspect is about, and align it more with the names of the models.